pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -295,12 +301,16 @@ class ErratumContentUnitSaver(ContentUnitSaver):
             update_record = declarative_content.content
             try:
                 update_collections = update_record._collections
+                update_references = update_record._references
             except AttributeError:
```

Comment:
I think this can just be dropped.

Reply:
I'm all for that but I want to understand the purpose of that try-except